src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs

```diff
@@ -73,11 +73,12 @@ public class VoiceCommand : IDisposable
         /// The public constructor.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// <exception cref="InvalidOperationException">
-        /// This can occur due to the following reasons:
-        /// 1) This exception can be due to an invalid state.
-        /// 2) This exception can be due to out of memory.
-        /// </exception>
+        /// <privilege> http://tizen.org/privilege/recorder </privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature> http://tizen.org/feature/microphone </feature>
+        /// <feature> http://tizen.org/feature/speech.control </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to out of memory.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
```

Comment:
wrong exception becuase VoiceCommand API has not argument

Reply:
Removed ArgumentException